socorro-cmd

```diff
@@ -49,6 +50,24 @@ def showcommands_cmd(argv):
         print('')
 
 
+def django_cmd(cmd):
+    """Run a Django command."""
+    def _django_cmd(argv):
+        argv = ['socorro-cmd', cmd] + argv
+        sys.path.insert(0, 'webapp-django')
+        import manage
+        manage.main(argv)
+    return _django_cmd
+
+
+def import_path(python_path):
+    """Import a name from a module."""
+    module_path, name = python_path.rsplit('.', 1)
+    __import__(module_path)
+    module = sys.modules[module_path]
+    return getattr(module, name)
```

Comment:
This code used to be in `import_and_run`. One nice thing about having it here is that it tries to import the destination whenever this command is run--not just when this specific subcommand is run. That's how I noticed that one of the scripts was dead. I think I like that better since `socorro-cmd` is manually tested.

Reply:
Oh, hot diggity! that's a new-in-Python-3 thing I didn't know about. I'll switch it.